content/en/llm_observability/instrumentation/sdk.md

```diff
@@ -1790,6 +1790,49 @@ def rag_workflow(user_question):
 
 [1]: /getting_started/tagging/
 
+{{% /tab %}}
+
+{{% tab "Node.js" %}}
+
+The SDK's `llmobs.annotationContext()` accepts a callback function that can be used to modify all auto-instrumented spans started while inside the scope of the callback function.
+
+The `llmobs.annotationContext()` method accepts the following options on the first argument:
+
+{{% collapse-content title="Options" level="h4" expanded=false id="annotating-autoinstrumented-span-arguments" %}}
+
+`name`
+: optional - _str_
+<br />Name that overrides the span name for any auto-instrumented spans that are started within the annotation context.
+
+`tags`
+: optional - _object_
+<br />An object of JSON serializable key-value pairs that users can add as tags on the span. Example keys: `session`, `env`, `system`, and `version`. For more information about tags, see [Getting Started with Tags][1].
```

Comment:
The "Getting Started with Tags" link is broken. I think numbered links don't work in collapsible content sections. Can you replace with an HTML link?

Reply:
:o this is broken for our python tab too - thanks for flagging! i'll replace both instances 😄